sdk/eventhub/event-hubs/tests.yml

```
@@ -21,50 +21,72 @@ jobs:
           TestType: "node"
           NodeVersion: "10.x"
           EVENTHUB_NAME: "hub_linux_node10"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         Windows_Node10:
           OSVmImage: "windows-2019"
           TestType: "node"
           NodeVersion: "10.x"
           EVENTHUB_NAME: "hub_windows_node10"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         macOS_Node10:
           OSVmImage: "macOS-10.13"
           TestType: "node"
           NodeVersion: "10.x"
           EVENTHUB_NAME: "hub_mac_node10"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         Linux_Node12:
           OSVmImage: "ubuntu-16.04"
           TestType: "node"
           NodeVersion: "12.x"
           EVENTHUB_NAME: "hub_linux_node12"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         Windows_Node12:
           OSVmImage: "windows-2019"
           TestType: "node"
           NodeVersion: "12.x"
           EVENTHUB_NAME: "hub_windows_node12"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         macOS_Node12:
           OSVmImage: "macOS-10.13"
           TestType: "node"
           NodeVersion: "12.x"
           EVENTHUB_NAME: "hub_mac_node12"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         Linux_Node8:
           OSVmImage: "ubuntu-16.04"
           TestType: "node"
           NodeVersion: "8.x"
           EVENTHUB_NAME: "hub_linux_node8"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         Windows_Node8:
           OSVmImage: "windows-2019"
           TestType: "node"
           NodeVersion: "8.x"
           EVENTHUB_NAME: "hub_windows_node8"
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
         macOS_Node8:
           OSVmImage: "macOS-10.13"
           TestType: "node"
           NodeVersion: "8.x"
           EVENTHUB_NAME: "hub_mac_node8"
-
+          IOTHUB_CONNECTION_STRING: $(js-event-hubs-test-iothub-connection-string)
+          EVENTHUB_CONNECTION_STRING: $(js-event-hubs-test-connection-string)
+        Browser_Windows:
```

```suggestion
        Windows_Browser:
```

The existing pattern is `OS_Runtime`